lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -21,8 +21,12 @@ class Msf::Ui::Console::CommandDispatcher::Developer
 
   def initialize(driver)
     super
-    output, status = modified_files
-    @modified_files = status.success? ? output : []
+    begin
+      output, status = modified_files
+      @modified_files = status.success? ? output : []
+    rescue => e
+      elog(e)
+    end
```

Comment:
This abstraction should belong in the `modified_files` implementation, otherwise crashes can still happen if someone calls `reload_lib -a` directly

Reply:
Very true, got that moved into `modified_files` 👍